doc/nrf/releases/release-notes-latest.rst

```diff
@@ -40,7 +40,14 @@ nRF9160
 nRF5
 ====
 
-There are no entries for this section yet.
+The following changes are relevant for the nRF52 and nRF53 Series.
+
+Bluetooth Low Energy
+--------------------
+
+* Updated:
+
+  * :ref:`ble_samples` - Changed bluetooth  sample name Central DFU SMP to :ref:`Central SMP Client <bluetooth_central_dfu_smp>`.
```

Comment:
I believe this should go under heading Applications and samples instead of nRF5.

Reply:
Fixed.

Reply:
So sorry, Marcin. I was wrong. It was in the right place already, after all.

Reply:
Modified. It is as before.